usaspending_api/disaster/v2/views/disaster_base.py

```diff
@@ -218,9 +217,31 @@ def pagination(self):
         return self.run_models(sortable_columns)
 
 
-class RecipientMixin:
-    required_filters = ["def_codes", "award_type_codes"]
+class ElasticsearchSpendingPaginationMixin(_BasePaginationMixin):
+    sum_column_mapping = {"obligation": "total_covid_obligation", "outlay": "total_covid_outlay"}
+    sort_column_mapping = {
+        "description": "_key",
+        "count": "_count",
+        **sum_column_mapping,
+    }
 
-    @property
-    def award_type_codes(self):
-        return self.filters.get("award_type_codes")
+    @cached_property
+    def pagination(self):
+        return self.run_models(list(self.sort_column_mapping), default_sort_column="description")
+
+
+class ElasticsearchLoansPaginationMixin(_BasePaginationMixin):
```

Comment:
These two probably make more sense to put in the `elasticsearch_base.py` module.